### PR TITLE
Update driver software

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR updates the Fedora and Arch sections for the system76 driver and software articles.

### CODE REVIEW

**Review by thomas-zimmerman — APPROVED** [review]
Looks good on Live preview.

File: content/system76-software.md [hunk 1]
```diff
@@ -157,6 +157,15 @@ paru -S system76-acpi-oled
 
 Be sure to install the <u>System76 Driver</u> first. The steps to do that are [here](/articles/system76-driver).
 
+This command will install all of the packages using <u>Paru</u>.
+
+```bash
+sudo dnf install system76* firmware-manager
```
Comment (XV-02) [inline content/system76-software.md:163]: Doesn't this do exactly what lines 169-180 do? (section "System76 Firmware Manager in Fedora")

File: content/system76-software.md [hunk 2]
```diff
@@ -167,6 +176,12 @@ sudo systemctl enable --now system76-firmware-daemon
 sudo gpasswd -a $USER adm
 ```
 
+**NOTE:** After enabling the dkms systemd service for any of the DKMS packages you will need to reboot the system:
```
Comment (XV-02) [inline content/system76-software.md:179]: The previous instructions don't cover DKMS packages?

**Review by XV-02 — CHANGES_REQUESTED** [review]
Some typo fixes needed, and also, the Fedora instructions in `content/system76-software.md` seem very out-of-order right now. We'll need to resolve that before we move forward.

File: content/system76-driver.md [hunk 3]
```diff
@@ -91,7 +67,7 @@ sudo pacman -S --needed base-devel git linux-headers
 These commands will clone, build and install the <u>System76 Firmware Daemon</u> service.
 
 ```bash
-git clone https://aur.archlinux.org/system76-firmware-daemon.git
+git clone https://aur.archlinux.org/system76-firmware-daemon-git.git
 cd system76-firmware-daemon
 makepkg -srcif
```
[the remark this answers is not in the document]
Comment (ahoneybun) [reply content/system76-driver.md:72]: We do not maintain the AUR package so we can't make this change, this works fine as it currently.

[the remark this answers is not in the document]
Comment (thomas-zimmerman) [reply content/system76-driver.md:72]: Double checking the live npm build this still looks good. While the update the PKGBUILD file would be nice, that will need to happen with the AUR maintainer.

**Review by thomas-zimmerman — DISMISSED** [review]
Looks good to merge

**Review by XV-02 — APPROVED** [review]

**Review by thomas-zimmerman — APPROVED** [review]
I would like to get these changes out. NPM live view looks good.

**Pull request merged**